python/cuml/metrics/cluster/silhouette_score.py

```diff
@@ -0,0 +1,72 @@
+import cupy as cp
+
+from cuml.metrics.cluster.silhouette_score_impl import _silhouette_coeff
+
+
+def silhouette_score(
+        X, labels, metric='euclidean', handle=None):
```

Comment:
small thing and something I had been talking with a few other devs, but one line per parameter (https://github.com/rapidsai/cuml/blob/b437b6614d074bf77c1bbc86ff3dde0f42ab8edd/python/cuml/common/input_utils.py#L204) seems like a worthwhile thing to standardize

Reply:
Sounds good; I'll open an issue for linting this as well.